packages/wrangler/src/proxy.ts

```diff
@@ -293,8 +295,8 @@ export function usePreviewServer({
 
     waitForPortToBeAvailable(port, { retryPeriod: 200, timeout: 2000 })
       .then(() => {
-        proxyServer.listen(port);
-        console.log(`⬣ Listening at ${localProtocol}://localhost:${port}`);
+        proxyServer.listen(port, ip);
+        console.log(`⬣ Listening at ${localProtocol}://${ip}:${port}`);
```

Comment:
could we show the ip here only if it isn't 127.0.0.1?

Reply:
Strictly `localhost` only refers to `127.0.0.1` if that is what is in the`/etc/hosts` (on OS/X) file.
Instead, how about we change the default for `ip` to be `localhost` rather than `127.0.01`, which would see more appropriate.

Reply:
Yeah that sounds good